frontend/libs/studio-components/src/components/StudioModal/StudioModal.tsx

```diff
@@ -52,7 +39,7 @@ export type ModalProps = {
  *
  * @returns {ReactNode} - The rendered component
  */
-export const Modal = ({ isOpen, onClose, title, children }: ModalProps): ReactNode => {
+export const StudioModal = ({ isOpen, onClose, title, children }: StudioModalProps): ReactNode => {
```

Comment:
Ref. forrige kommentar, så ser jeg for meg at vi kan oppdatere denne med å utvide typen med native-elementet og sende forwardRef? Hva tenker du? 😊

Reply:
Yes 😄